config/initializers/devise.rb

```diff
@@ -249,8 +249,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  Devise.setup do |config|
-    config.omniauth :facebook, ENV["FB_ID"], ENV["FB_SECRET"],
+  Devise.setup do |var|
+    var.omniauth :facebook, ENV["FB_ID"], ENV["FB_SECRET"],
```

Comment:
Not sure what I have to do here. I would need to study properly how omniauth works (I was not the one who wrote this code). I would propose to remove everything related with omniauth and implement this later in a new commit.